website/docs/dev/cluster/migration.md

```diff
@@ -0,0 +1,310 @@
+---
+id: slot-migration
+sidebar_label: Migration
+title: Migration Overview
+---
+
+# Slot Migration Overview
+
+In Garnet, slot migration, refers to the act of re-assigning slot ownernship and moving (i.e. ***migrating***) the associated key-values pairs from one node to another while the cluster is fully operational.
```

Comment:
This might clarify this sentence:

```suggestion
In Garnet, slot migration involves re-assigning slot ownership from one node to another and moving (i.e., ***migrating***) the associated key-values pairs while the cluster is fully operational.
```

Reply:
Rewrote, thanks for the suggestion!